### PR TITLE
Added README, SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Added README, SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING files.

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
-# cortx-utils
-
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/f2e9061b01884d0f920adfecf6117024)](https://app.codacy.com/gh/Seagate/cortx-utils?utm_source=github.com&utm_medium=referral&utm_content=Seagate/cortx-utils&utm_campaign=Badge_Grade)
 
-CORTX Utility code used by various components
+# CORTX-Utils
+
+The cortx-utils repository contains utility tools and code for various CORTX submodules. For example, S3 and NFS submodules use code from the cortx-utils repos.
```
Comment (sachinpunadikar) [inline README.md:5]: The cortx-utils repository contains utility tools and code for various components in CORTX stack. For example, S3 and NFS components use code from the cortx-utils repo.

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Look at the codacy issues and other suggestions

**Comment by sachinpunadikar** [comment]
@Saumya-Sunder Please work on the suggested change.

**Review by s-arya — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**